src/Requests.php

```diff
@@ -224,49 +224,78 @@ public static function add_transport($transport) {
 	}
 
 	/**
-	 * Get a working transport
+	 * Get the FQCN for a working transport.
```

Comment:
```suggestion
	 * Get the fully qualified class name (FQCN) for a working transport.
```

(Lower the threshold for entry for less experienced contributors and users of the API docs)